python/sglang/srt/constrained/xgrammar_backend.py

```diff
@@ -86,20 +84,19 @@ def jump_and_retokenize(
     def allocate_vocab_mask(
         self, vocab_size: int, batch_size: int, device
     ) -> torch.Tensor:
-        return self.matcher.allocate_token_bitmask(vocab_size, batch_size)
+        return allocate_token_bitmask(vocab_size, batch_size)
```

Comment:
allocate_token_bitmask(batch_size, vocab_size)

Reply:
Blunder. Thank you.